HW-03/main.py

```diff
@@ -0,0 +1,33 @@
+text = 'I love Python '           
+text *= 42
+print(text)
+
+
+one_month_of_the_year = 10 / 12   # Для вирахування  значення одного місяця в році
+age = 23                         
+month_of_year = 12               
+remaining_months = 5             
+age_in_month = age * month_of_year +(remaining_months * one_month_of_the_year)
+print(age_in_month)
+
+age_in_years = age
```

Comment:
Age in years should be calculated from `age_in_month` variable. You can use operator of division for it